packages/flutter/lib/src/gestures/events.dart

```diff
@@ -2099,28 +2099,28 @@ class PointerCancelEvent extends PointerEvent with _PointerEventDescription, _Co
 }
 
 /// Determine the appropriate hit slop pixels based on the [kind] of pointer.
-double computeHitSlop(PointerDeviceKind kind) {
+double computeHitSlop(PointerDeviceKind kind, double? deviceTouchSlop) {
```

Comment:
should these just take a DeviceGestureSettings obj and the method can pick the appropriate value from it?